### PR TITLE
glitter: Add version 1.0.0

### PR DESCRIPTION
**Pull request by Milo123459**
Adds support for glitter!

Note I wasn't sure on how to get the hash so that is the commit hash. Please let me know of how to get it :)

### CODE REVIEW

File: bucket/glitter.json [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+        },
+        "32bit": {
+            "url": "https://github.com/Milo123459/glitter/releases/download/v1.0.0/glitter-i686-pc-windows-msvc.zip",
+            "hash": "83a8bfa0e7ec2cbde05c27c51e6ccff1ce376db2"
+        }
+    },
+    "bin": "glitter.exe",
+    "checkver": {
+        "github": "https://github.com/Milo123459/glitter"
+    },
```
Comment (r15ch13) [inline bucket/glitter.json:19]:
```suggestion
	"checkver": "github",
```

File: bucket/glitter.json [hunk 2]
```diff
@@ -0,0 +1,33 @@
+{
+    "version": "1.0.0",
+    "description": "Git tooling of the future",
+    "homepage": "https://github.com/Milo123459/glitter",
+    "license": "MIT",
+    "architecture": {
+        "64bit": {
+            "url": "https://github.com/Milo123459/glitter/releases/download/v1.0.0/glitter-x86_64-pc-windows-msvc.zip",
+            "hash": "83a8bfa0e7ec2cbde05c27c51e6ccff1ce376db2"
```
Comment (r15ch13) [inline bucket/glitter.json:9]: These should be the checksum of the file itself, not the commit sha1.

Comment (Milo123459) [reply bucket/glitter.json:9]: alright

**Comment by Milo123459** [comment]
Sorry about this again, but if I wanted to add this to brew, how can I obtain the checksum?

**Comment by r15ch13** [comment]
I don't know if brew has something that gets the checksum automatically.

https://docs.brew.sh/Adding-Software-to-Homebrew
https://docs.brew.sh/Formula-Cookbook

**Review by r15ch13 — APPROVED** [review]

**Pull request merged**